### ISSUE TITLE
ButtonBar MultiStack left side?

### ISSUE THREAD

**Issue by Allan-Nava** [rokudev/SceneGraphDeveloperExtensions#74]
Is possible to have the buttonbar of navigation on the left

**Comment by RokuChris** [comment]
Yes it is. just set the _alignment_ field on the buttonbar to "left"

https://github.com/rokudev/SceneGraphDeveloperExtensions/blob/master/documentation/1-components.md#buttonbar

**Comment by Allan-Nava** [comment]
Ok now I try, like this:

![2021-07-19 17 43 47](https://user-images.githubusercontent.com/22498435/126188178-eda58587-19af-4a23-a5eb-41cf6ddc497a.jpg)

**Comment by Allan-Nava** [comment]
Perfect it works! but it is possible to customize it?

**Comment by Allan-Nava** [comment]
Any news to customize the sidebar?

**Comment by RokuChris** [comment]
The current options for customizing ButtonBar are documented here: https://github.com/rokudev/SceneGraphDeveloperExtensions/blob/master/documentation/1-components.md#buttonbar

If there are other specific features you'd like to see we're happy to consider them.

**Comment by Allan-Nava** [comment]
Ok i customized the button bar item 

![telegram-cloud-photo-size-4-5847991835346187899-y](https://user-images.githubusercontent.com/22498435/130576550-169a2023-35ea-4ecc-bfbd-1fe2a00ff72e.jpg)
